### PR TITLE
Updating URL validation

### PR DESCRIPTION
**Pull request by karthikeyan-cere**
Adding additional validation for URL using regex

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## VNext 
+- Updating URL validation
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md:4]: The description should be in a completed form. Not `Updating` but `Updated`.

File: packages/apps/src/Endpoints/index.tsx [hunk 2]
```diff
@@ -35,11 +35,15 @@ interface UrlState {
...
 
 function isValidUrl (url: string): boolean {
+  const regex = /^(wss?:\/\/)([0-9]{1,3}(?:\.[0-9]{1,3}){3}|[^\/]+):([0-9]{4,5})$/;
+
   return (
     // some random length... we probably want to parse via some lib
     (url.length >= 7) &&
     // check that it starts with a valid ws identifier
-    (url.startsWith('ws://') || url.startsWith('wss://'))
+    (url.startsWith('ws://') || url.startsWith('wss://')) &&
```
Comment (AndreiNavoichyk) [inline packages/apps/src/Endpoints/index.tsx:44]: Include these checks to the regex.

File: packages/apps/src/Endpoints/index.tsx [hunk 3]
```diff
@@ -35,11 +35,15 @@ interface UrlState {
 const STORAGE_AFFINITIES = 'network:affinities';
 
 function isValidUrl (url: string): boolean {
+  const regex = /^(wss?:\/\/)([0-9]{1,3}(?:\.[0-9]{1,3}){3}|[^\/]+):([0-9]{4,5})$/;
+
   return (
     // some random length... we probably want to parse via some lib
     (url.length >= 7) &&
```
Comment (AndreiNavoichyk) [inline packages/apps/src/Endpoints/index.tsx:42]: Remove this check

**Review by AndreiNavoichyk — COMMENTED** [review]
Tests should be added.

File: CHANGELOG.md (hunk 1 above)
Comment (karthikeyan-cere) [reply CHANGELOG.md:4]: Done

**Review by AndreiNavoichyk — COMMENTED** [review]
Linter check became **red**. Please, check it.

File: packages/apps/src/Endpoints/index.tsx (hunk 3 above)
Comment (karthikeyan-cere) [reply packages/apps/src/Endpoints/index.tsx]: Done.

File: packages/apps/src/Endpoints/index.tsx (hunk 2 above)
Comment (karthikeyan-cere) [reply packages/apps/src/Endpoints/index.tsx]: Done

**Review by AndreiNavoichyk — COMMENTED** [review]
Tests should be added.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 # CHANGELOG
 
-## VNext
+
+## vNext 
+- Updated URL validation
```
Comment (AndreiNavoichyk) [inline CHANGELOG.md:5]: Let's keep the right order here. This update should be in the second line as far as `Added assert` is already in the `dev`.

Comment (karthikeyan-cere) [reply CHANGELOG.md]: Updated

**Pull request merged**